pootle/apps/reports/views.py

```diff
@@ -204,6 +204,12 @@ def get_detailed_report_context(user, month):
                 else:
                     totals['reviewed'][score.review_rate] = {'words': reviewed}
 
+            suggested = score.get_suggested_words()
+            if suggested is not None:
+                score.action = _('Suggestion')
```

Comment:
The main concern is that `score` shouldn't be changed. I think julen comment is quite clear.